File: src/edu/gbcg/dbInteraction/dbcreator/IndexWorker.java
package edu.gbcg.dbInteraction.dbcreator;

import edu.gbcg.dbInteraction.DBCommon;

import java.sql.Connection;

public class IndexWorker implements Runnable{
    Connection conn;
    String SQLStatement;

    public IndexWorker(Connection conn, String SQLStatement){
        this.conn = conn;
        this.SQLStatement = SQLStatement;
    }

    public void run(){
        DBCommon.execute(conn, SQLStatement);
    }
}

File: src/main/java/edu/gbcg/dbInteraction/dbcreator/IndexWorker.kt
/*
 * Copyright (c) 2018 Sean Grimes. All Rights Reserved.
 * License: MIT
 */

package edu.gbcg.dbInteraction.dbcreator

import edu.gbcg.dbInteraction.DBCommon
import java.sql.Connection

class IndexWorker(conn: Connection, SQLStatement: String): Runnable{
    private val conn = conn
    private val SQLStatement = SQLStatement

    override fun run(){ DBCommon.execute(conn, SQLStatement) }
}